diff --git a/node_modules/@gorhom/animated-tabbar/lib/commonjs/utilities/reanimated.js b/node_modules/@gorhom/animated-tabbar/lib/commonjs/utilities/reanimated.js
index 147ddf9..91b290d 100644
--- a/node_modules/@gorhom/animated-tabbar/lib/commonjs/utilities/reanimated.js
+++ b/node_modules/@gorhom/animated-tabbar/lib/commonjs/utilities/reanimated.js
@@ -6,18 +6,15 @@ Object.defineProperty(exports, "__esModule", {
 exports.interpolate = exports.Easing = void 0;
 
 const {
-  Easing: EasingV1,
   EasingNode: EasingV2
 } = require('react-native-reanimated');
 
-const Easing = EasingV2 || EasingV1;
-exports.Easing = Easing;
+exports.Easing = EasingV2;
 
 const {
-  interpolate: interpolateV1,
   interpolateNode: interpolateV2
 } = require('react-native-reanimated');
 
-const interpolate = interpolateV2 || interpolateV1;
+const interpolate = interpolateV2;
 exports.interpolate = interpolate;
 //# sourceMappingURL=reanimated.js.map
\ No newline at end of file
