diff --git a/node_modules/geojson-places/src/index.js b/node_modules/geojson-places/src/index.js
index 6f5a9b9..fd2484a 100644
--- a/node_modules/geojson-places/src/index.js
+++ b/node_modules/geojson-places/src/index.js
@@ -1,10 +1,10 @@
-const fs = require('fs');
-const path = require('path');
+// const fs = require('fs');
+// const path = require('path');
 const clone = require('just-clone');
 const { pointInPolygon, translateNames, translateName } = require('./utils');
 const { admin1, continents, countries, countryGroupings, regions } = require('./data');
 
-const dataPath = path.join(__dirname, './../data/');
+// const dataPath = path.join(__dirname, './../data/');
 
 /**
  * Get information from coordinates
@@ -80,24 +80,24 @@ const lookUpGeoJSON = (lat, lon) => {
 };
 
 const getContinentGeoJSONByCode = (continent_code, simplified = false) => {
-    try {
-        const filePath = `/continents/${continent_code}${simplified ? '-simplified' : ''}.json`;
-        const continent = fs.readFileSync(path.join(dataPath, filePath));
-        return JSON.parse(continent);
-    } catch(e) {
-        console.error(e);
+    // try {
+    //     const filePath = `/continents/${continent_code}${simplified ? '-simplified' : ''}.json`;
+    //     const continent = fs.readFileSync(path.join(dataPath, filePath));
+    //     return JSON.parse(continent);
+    // } catch(e) {
+    //     console.error(e);
         return null;
-    }
+    // }
 };
 
 const getCountryGeoJSONByAlpha2 = (alpha2) => {
-    try {
-        const country = fs.readFileSync(path.join(dataPath, `/countries/${alpha2}.json`));
-        return JSON.parse(country);
-    } catch(e) {
-        console.error(e);
+    // try {
+    //     const country = fs.readFileSync(path.join(dataPath, `/countries/${alpha2}.json`));
+    //     return JSON.parse(country);
+    // } catch(e) {
+    //     console.error(e);
         return null;
-    }
+    // }
 };
 
 const getCountryGeoJSONByAlpha3 = (alpha3) => {
@@ -106,24 +106,24 @@ const getCountryGeoJSONByAlpha3 = (alpha3) => {
 };
 
 const getCountryGroupingGeoJSONByCode = (grouping_code, simplified = false) => {
-    try {
-        const filePath = `/country-groupings/${grouping_code}${simplified ? '-simplified' : ''}.json`;
-        const countryGrouping = fs.readFileSync(path.join(dataPath, filePath));
-        return JSON.parse(countryGrouping);
-    } catch(e) {
-        console.error(e);
+    // try {
+    //     const filePath = `/country-groupings/${grouping_code}${simplified ? '-simplified' : ''}.json`;
+    //     const countryGrouping = fs.readFileSync(path.join(dataPath, filePath));
+    //     return JSON.parse(countryGrouping);
+    // } catch(e) {
+    //     console.error(e);
         return null;
-    }
+    // }
 };
 
 const getRegionGeoJSONByCode = (region_code) => {
-    try {
-        const region = fs.readFileSync(path.join(dataPath, `/regions/${region_code}.json`));
-        return JSON.parse(region);
-    } catch(e) {
-        console.error(e);
+    // try {
+    //     const region = fs.readFileSync(path.join(dataPath, `/regions/${region_code}.json`));
+    //     return JSON.parse(region);
+    // } catch(e) {
+    //     console.error(e);
         return null;
-    }
+    // }
 };
 
 const getStateGeoJSONByCode = (state_code) => {
