diff --git a/node_modules/react-native-geojson/index.js b/node_modules/react-native-geojson/index.js
index e2e6984..9fe74f6 100644
--- a/node_modules/react-native-geojson/index.js
+++ b/node_modules/react-native-geojson/index.js
@@ -1,5 +1,4 @@
 import React, { Component, PropTypes } from 'react';
-import { View } from 'react-native';
 import MapView from 'react-native-maps';
 import uuid from 'uuid';
 
@@ -78,12 +77,15 @@ const makeCoordinates = feature => {
 
 const Geojson = props => {
   const overlays = makeOverlays(props.geojson.features);
+
+  // Fix from here: https://github.com/frankrowe/react-native-geojson/pull/6/files
   return (
-    <View>
+    <React.Fragment key={props.geojson.id || uuid()}>
       {overlays.map(overlay => {
         if (overlay.type === 'point') {
           return (
             <MapView.Marker
+              {...props}
               key={overlay.id}
               coordinate={overlay.coordinates}
               pinColor={props.color}
@@ -93,6 +95,7 @@ const Geojson = props => {
         if (overlay.type === 'polygon') {
           return (
             <MapView.Polygon
+              {...props}
               key={overlay.id}
               coordinates={overlay.coordinates}
               holes={overlay.holes}
@@ -105,6 +108,7 @@ const Geojson = props => {
         if (overlay.type === 'polyline') {
           return (
             <MapView.Polyline
+              {...props}
               key={overlay.id}
               coordinates={overlay.coordinates}
               strokeColor={props.strokeColor}
@@ -113,7 +117,7 @@ const Geojson = props => {
           );
         }
       })}
-    </View>
+    </React.Fragment>
   );
 };
 
