diff --git a/node_modules/react-native-map-clustering/lib/ClusteredMapView.js b/node_modules/react-native-map-clustering/lib/ClusteredMapView.js
index 1e35c99..7f2a453 100644
--- a/node_modules/react-native-map-clustering/lib/ClusteredMapView.js
+++ b/node_modules/react-native-map-clustering/lib/ClusteredMapView.js
@@ -5,6 +5,7 @@ import React, {
   useMemo,
   useRef,
   forwardRef,
+  useImperativeHandle,
 } from "react";
 import { Dimensions, LayoutAnimation, Platform } from "react-native";
 import MapView, { Marker, Polyline } from "react-native-maps";
@@ -58,6 +59,11 @@ const ClusteredMapView = forwardRef(
     const [clusterChildren, updateClusterChildren] = useState(null);
     const mapRef = useRef();
 
+
+    useImperativeHandle(ref, () => ({
+      mapView: mapRef.current
+    }));
+
     const propsChildren = useMemo(() => React.Children.toArray(children), [
       children,
     ]);
